arangod/Graph/ShortestPathOptions.h

```diff
@@ -109,6 +111,16 @@ struct ShortestPathOptions : public BaseOptions {
   auto setDefaultWeight(double weight) -> void;
   auto getDefaultWeight() const -> double;
 
+  auto setDisjoint() -> void;
+  auto isDisjoint() const -> bool;
+
+  auto isSatelliteLeader() const -> bool;
+
+  bool checkSmartDestination(VPackSlice edge,
+                             std::string_view sourceVertex) const;
```

Comment:
Are these implemented? The declarations appear new in the header file, but I do not see a corresponding implementations in the CPP (for the enterprise case).

Reply:
Yes they are.